backend/Utils/EmailUtils.ts

```diff
@@ -2,7 +2,7 @@ import google from "googleapis";
 import path from "path";
 import { encode } from "js-base64";
 
-export const baseUrl = process.env.SERVER_ORIGIN || "https://localhost:7600";
+export const baseUrl = process.env.SERVER_ORIGIN || "http://localhost:7600";
```

Comment:
Since this environment variable isn't a security concern, I'm fine with having a default value here - a hacker would not benefit from seeing the default value